server/middlewares/auth.js

```diff
@@ -0,0 +1,45 @@
+const {User} = require(`../models`)
+const {decode} = require(`../helpers/jwt`)
+
+const authentication = (req, res, next) => {
+    let access_token = req.headers.access_token
+    // console.log(access_token, `ini di auth`)
+    let userData = decode(access_token)
+    req.user = userData
+    User.findByPk(userData.id)
+    .then(result => {
```

Comment:
btw sudah dicoba kah, sekiranya user not found, kehandle kah errornya? `if(!result)` ?

Reply:
sudah kakk, karna access_tokennya jatohnya gaada/invalid masuk ke error 401